_departmentpeople/gradstudents/sm3vg.md

```diff
@@ -8,7 +8,7 @@ office: 110 Kerchof Hall
 phone: 434-924-4947
 email: sm3vg@virginia.edu
 image: __SITE_URL__/img/people/2019-fall/Maitra.jpg
-personal_page: https://people.virginia.edu/~sm3vg/
```

Comment:
people.virginia.edu pages are going to disappear in December as ITS mentioned in its email